js/chirp.js

```diff
@@ -1,7 +1,11 @@
-function chirp(n){
-  // FIX ME
+function chirp(n) {
+  let x = "chirp ";
+  if (n === 1) {
+    return x;
+  } else {
+    return (x += chirp(--n));
```

Comment:
Can you think of a way to reformat this so that the last chirp doesn't have a space at the end?